CHANGELOG.md

```diff
@@ -5,6 +5,143 @@
+- [pickers] Remove  `FieldValueType` in favor of `PickerValueType` (#15259) @arthurbalduini
+- [pickers] Remove the form props from the layout and the toolbar slots (#15492) @flaviendelangle
+- [pickers] Use `props.referenceDate` timezone when `props.value` and `props.defaultValue` are not defined (#15532) @flaviendelangle
+- [TimePicker] Prevent mouse events after `touchend` event (#15346) @arthurbalduini
+
+#### `@mui/x-date-pickers-pro@v8.0.0-alpha.1` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@v8.0.0-alpha.1`, plus:
+
+- [DateTimeRangePicker] Use time in `referenceDate` when selecting date (#15429) @LukasTy
+
+### Charts
+
+#### Breaking change
+
+- The DX of the Tooltip customization has been refactored
+  - The `tooltip` prop is removed in favor of `slotProps.tooltip` for consistency.
+  - The previous slots `popper`, `axisContent`, `itemContent` are removed in favor of a `tooltip` slots which override the entire tooltip.
+    - To override the tooltip content, use the `useItemTooltip` or `useAxisTooltip` to get the data, and wrapp your component in `ChartsTooltipContainer` to follow the pointer position.
+    - To override the tooltip placement, use the `ChartsItemTooltipContent` or `ChartsItemTooltipContent` to get default data and place them in your custom tooltip.
+
+- The library now uses the SVG `filter` attribute instead of `d3-color` for color manipulation.
+  - This modification impacts the `LinePlot`, `AreaPlot`, and `BarPlot`.
+    If you've customized the `fill` of those elements, you might need to override it by using the CSS `filter`.
+  - The `theme.styleOverride` is removed for `MuiLineElement`, `MuiAreaElement`, and `MuiBarElement` to improve performances.
+    You can still target those elements by using the `MuiLinePlot`, `MuiAreaPlot`, and `MuiBarPlot` and target the appropriate classes `lineElementClasses.root`, `areaElementClasses.root`, `barElementClasses.root`
+
+- Removed the `resolveSizeBeforeRender` prop from all charts component — [Learn more](https://next.mui.com/x/migration/migration-charts-v7/#remove-resolvesizebeforerender-prop)
+- Removed `width` and `height` props from the `ChartsSurface` component.
+- Removed the `viewport` prop from all charts.
+
+#### `@mui/x-charts@v8.0.0-alpha.1`
+
+- [charts] Decouple `ChartDataProvider` and `ChartsSurface` (#15375) @JCQuintas
+- [charts] Fix `ScatterChart` tooltip wrong defaults (#15537) @JCQuintas
+- [charts] Fix key generation for the ChartsGrid (#15463) @alexfauquette
+- [charts] Improve `SvgRefProvider` to split the received ref (#15424) @JCQuintas
+- [charts] Move interaction state in store (#15426) @alexfauquette
+- [charts] Refactor Tooltip customisation (#15154) @alexfauquette
+- [charts] Remove intrinsic size requirement (#15471) @JCQuintas
+- [charts] Replace d3-color by CSS filter for highlight (#15084) @alexfauquette
```

Comment:
```suggestion
- [charts] Replace `d3-color` with CSS filter for highlight (#15084) @alexfauquette
```

Seems to be more common in active form: https://english.stackexchange.com/questions/151309/replace-with-versus-replace-by